src/Symfony/Component/HttpFoundation/Tests/ProblemJsonResponseTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ */
+
+namespace Symfony\Component\HttpFoundation\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\ProblemJsonResponse;
+
+/**
+ * @author Abdellah Ramadan <ramadanabdel24@gmail.com>
+ */
+class ProblemJsonResponseTest extends TestCase
+{
+    public function testNewProblemWithNoParams()
+    {
+        $problemDetails = new ProblemJsonResponse();
+        $this->assertSame(500, $problemDetails->getStatusCode());
+
+        $this->assertSame('about:blank', json_decode($problemDetails->getContent(), true)['type']);
+
```

Comment:
```suggestion
```

Reply:
@OskarStark please could this be merged already?
I fixed the highlighted problems.